doc/May-2022-Upgrade.md

```diff
@@ -0,0 +1,634 @@
+## Upgrading from an old version of kobo-docker (before May 2022)
+
+Current versions of kobo-docker require PostgreSQL 14, MongoDB 5 and Redis 6
+
+
+If you are running a version of kobo-docker that was last updated prior to
+May 2022 (i.e. commit TBC or older),
+you need to upgrade your databases prior to using the current version of
+kobo-docker (this repository) or
+[kobo-install](https://github.com/kobotoolbox/kobo-install).
+
+This is a step-by-step procedure to upgrade PostgreSQL and MongoDB.
+
+**This procedure has been tested on x86 architecture only.**
+
+### PostgreSQL
+
+**Be sure to have enough space left on the host filesystem before upgrading.**
+Check the size of the PostgreSQL database in  `.vols/db`, e.g. with
+`sudo du -hs .vols/db`, and ensure you have _more_ than this amount of space
+free.
+
+For this tutorial, we are using kobo-install to run docker-compose commands.
+If you do not use kobo-install, please replace `python run.py -cb` with `docker-compose -f docker-compose.primary.backend.template.yml -f docker-compose.primary.backend.yml [-f docker-compose.primary.backend.override.yml] [-f docker-compose.primary.backend.custom.yml]`
+
+1. Stop the containers
+
+    ```shell
+    user@computer:kobo-install$ python run.py --stop  
+    ```
```

Comment:
I would suggest changing these commands to something that's easily copied by clicking the copy button in GitHub:

> From within the kobo-install directory
```
python3 run.py --stop
```
or
```
./run.py --stop
```